opba-banking-rest-api/src/main/resources/static/tpp_banking_api_ais.yml

```diff
@@ -68,6 +71,7 @@ paths:
         - $ref: "#/components/parameters/Fintech-Redirect-URL-OK"
         - $ref: "#/components/parameters/Fintech-Redirect-URL-NOK"
         - $ref: "./tpp_banking_api_commons.yml#/components/parameters/PSU-Consent-Session"
+        - $ref: "./tpp_banking_api_commons.yml#/components/parameters/Service-Session-ID"
```

Comment:
To my understanding this shall replace `PSU-Consent-Session`. TBD with Francis

Reply:
Yes it should, we need to have clarification session